components/properties/src/trievalue.rs

```diff
@@ -97,6 +99,14 @@ impl TrieValue for SentenceBreak {
     }
 }
 
+impl TrieValue for BidiPairedBracketType {
```

Comment:
Nit: Don't add these impls unless we need them for something, and I _think_ we don't need it for this one

Reply:
Well, now that we're getting strict again with this being a closed enum, how about you move this type to be defined directly in the datagen crate, if that is the only user?

Reply:
Inside the implementation of one of the APIs in the properties crate, we need to match the `CheckedBidiPairedBracketType` enum value in order to return the correct value for the struct.  (`BidiAuxiliaryPropertiesBorrowed::get32_pairing_props`)